frontend-react/src/components/annotate.js

```diff
@@ -3,73 +3,164 @@ import ReactImageAnnotate from "react-image-annotate";
+      });
+
+      //If there are no more images left in Annotator
+      if (this.state.index < 0){
+        //Set index back to 0
+        this.setState({
+          index: this.state.index = 0
+        });
       }
+    }
+    //Cancel deletion
+    else{
+      return
+    }
+  }
+
+  incrementIndex(imagesIndex){
+    this.setState({
+      index: this.state.index = imagesIndex.length
+    });
+  }
+
+  //Each time the annotator is instantiated, the current index is saved
+  updateIndex(index){
+    this.setState({
+      index: this.state.index=index.selectedImage
+    });
+  }
+
+  saveAnnotation(annotateData){
+    //Save current images and annotations to state data
+    this.setState({
+      data: annotateData.images,
+      saved: true
+    });
+
+
+  }
+
+  //Open file input dialog
+  clickFileInput(){
+
+    var element = document.querySelector('#root > div > form > label')
+    const mouseClickEvents = ['mousedown', 'click', 'mouseup'];
+    mouseClickEvents.forEach(mouseEventType =>
+      element.dispatchEvent(
+        new MouseEvent(mouseEventType, {
+            view: window,
+            bubbles: true,
+            cancelable: true,
+            buttons: 1
+        })
+      ),
+    );
+  }
+
+  render() {
+    let {data,
+          uploaded,
+          index
+        } = this.state;
+
+    //If annotator has images uploaded
+    if(uploaded && data.length > 0){
+      return (
+      <div className="border">
+
+        <form className="selectForm">
+            <input type="file" id="file" multiple={true} accept="image/*" className="fileInputButton" onChange={this.fileSelectedHandler} />
+            <label htmlFor="file" className="fileLabel">
+              Add Images
+            </label>
+        </form>
+
+        <div className="imageDisplay">
+
+          <ReactImageAnnotate
+            images= {data}
+            key = {data}
+            updateIndex = {output => this.updateIndex(output.selectedImage)}
+            regionClsList={[]}
+            enabledTools="create-polygon, create-box"
+            deleteImage={output => this.delete(output)}
+            addImages={output => {
+              this.saveAnnotation(output);
+              this.clickFileInput();
+            }}
+            selectedImage = {index}
+            onExit={output => console.log(JSON.stringify(output.images))}
+          />
+
+        </div>
+      </div>
+    );}
+    //If no images have been uploaded show "upload images" button
+    else{
+      return (
+        <div className="border">
+          <div className="imageDisplay">
+
+            <form className="selectForm" id="formUpload">
+                <input type="file" id="file" multiple={true} accept="image/*" className="fileInputButton" onChange={this.fileSelectedHandler} />
+                <label htmlFor="file" className="fileLabel">
+                    Upload Images
+                </label>
+            </form>
+
+          </div>
+        </div>
```

Comment:
Same as above in terms of the ids etc.

An idea for this is that this is removed and a base sample image is added to the annotator showing some instructions on how to use. Then the "add image" button is clicked which clears the sample image and shows the users images.